event/event.yaml

```diff
@@ -3,71 +3,55 @@ components:
     Event:
       description: |-
         The optional container for event configuration.
+        To get cp-dp data both cp_events and dp_events must be explicitly enabled. 
       type: object
       properties:
-        enable:
+        cp_events:
           description: |-
-            True to enable all events. 
-            Enabling this option may affect the resultant packet payload due to 
-            additional instrumentation data.
+            The optional container for control plane(cp) event configuration.
+          $ref: '#/components/schemas/Event.CPEvents'
           type: boolean
           default: false
           x-field-uid: 1
-        link:
-          $ref: '#/components/schemas/Event.Link'
-          x-field-uid: 2
-        rx_rate_threshold:
-          $ref: '#/components/schemas/Event.RxRateThreshold'
-          x-field-uid: 3
-        route_advertise_withdraw:
-          $ref: '#/components/schemas/Event.RouteAdvertiseWithdraw'
-          x-field-uid: 4
-    Event.RxRateThreshold:
-      description: |-
-        The optional container for rx rate threshold event configuration.
-      type: object
-      properties:
-        enable:
+        dp_events:
           description: |-
-            True to enable the rx_rate_threshold event. 
+            The optional container for data plane(dp) event configuration.
             Enabling this option may affect the resultant packet payload due to 
             additional instrumentation data.
-          type: boolean
-          default: false
-          x-field-uid: 1
-        threshold:
-          description: |-
-            True to enable notifications when the rx rate of a flow passes above 
-            or below the threshold value. 
-          type: number
-          format: float
-          maximum: 100
-          minimum: 0
-          default: 95
+          $ref: '#/components/schemas/Event.DPEvents'
           x-field-uid: 2
-    Event.Link:
+    Event.CPEvents:
       description: |-
-        The optional container for link up/down event configuration.
+        The optional container for control plane event configuration.
       type: object
       properties:
         enable:
           description: |-
-            True to enable notifications when a link up/down event occurs. 
+            True to enable all cp events.
           type: boolean
           default: false
           x-field-uid: 1
-    Event.RouteAdvertiseWithdraw:
+    Event.DPEvents:
       description: |-
-        The optional container for route advertise/withdraw event configuration.
+        The optional container for data plane event configuration.
       type: object
       properties:
         enable:
           description: |-
-            True to enable notifications when a route advertise/withdraw 
-            event occurs. 
+            True to enable all dp events.
```

Comment:
Setting to true enables flow_rx_rate_above_threshold and flow_rx_rate_below_threshold timestamps to be recorded when  data packets switch between multiple rx_ports on the receive ports for affected flows.